display_information/update-graphics/src/main/java/com/esri/samples/update_graphics/UpdateGraphicsSample.java

```diff
@@ -51,9 +54,7 @@
 
 public class UpdateGraphicsSample extends Application {
 
-  private boolean isUpdateLocationActive;
   private List<SimpleMarkerSymbol> markers;
-  private Button updateLocationButton;
   private Button updateDescriptionButton;
   private ComboBox<String> symbolBox;
 
```

Comment:
@Rachael-E on the line below we have the comment regarding keeping `map` in scope to avoid garbage collection. I can't see any part of the code where this might happen here? The only async operation is on `mapView`:
```
mapView.identifyGraphicsOverlayAsync
```

Was it decided to leave these comments in as a standard, or can we remove this one and make `map` a local member?